bucket/tunnel.json

```diff
@@ -1,39 +1,33 @@
 {
-    "version": "0.2.12",
+    "version": "0.3.2",
     "homepage": "https://github.com/labstack/tunnel-client",
-    "description": "Expose local servers to internet securely.",
+    "description": "Expose local servers to the internet securely.",
     "license": "MIT",
     "architecture": {
         "32bit": {
-            "url": "https://github.com/labstack/tunnel-client/releases/download/v0.2.12/tunnel-client_0.2.12_windows_32-bit.zip",
-            "hash": "bb5af7de73e6636f3e1cfe07af8be251f562248f9dac21eb886079d20893c10c"
+            "url": "https://github.com/labstack/tunnel-client/releases/download/v0.3.2/tunnel_0.3.2_windows_32-bit.zip",
+            "hash": "98d9bf222c817103bcf52a6428ca1746fed502bcc9b9950d52c275d8be3b1bab"
         },
         "64bit": {
-            "url": "https://github.com/labstack/tunnel-client/releases/download/v0.2.12/tunnel-client_0.2.12_windows_64-bit.zip",
-            "hash": "0923cb853726b9d62ab1b9cf9a1b18b2e5e0705fd3896143e1da948b0a8cf6eb"
+            "url": "https://github.com/labstack/tunnel-client/releases/download/v0.3.2/tunnel_0.3.2_windows_64-bit.zip",
+            "hash": "e1c8635bbee3dd4df00324deb295779e3e1f2ab046b4fa0a8bcd8f22cf5f8bbe"
         }
     },
-    "bin": [
-        [
-            "tunnel-client.exe",
-            "tunnel"
-        ],
-        "tunnel-client.exe"
-    ],
+    "bin": "tunnel.exe",
     "checkver": {
```

Comment:
```suggestion
    "checkver": "github",
```

Reply:
@Ash258 Looks like something wrong in this change, can you review again?